tests/core_tests.py

```diff
@@ -173,7 +173,6 @@ def test_viz_cache_key(self):
         viz = slc.viz
         qobj = viz.query_obj()
         cache_key = viz.cache_key(qobj)
-        self.assertEqual(cache_key, viz.cache_key(qobj))
```

Comment:
This seems unnecessary, i.e., it's equivalent to, 

```python
self.assertEqual(viz.cache_key(qobj), viz.cache_key(qobj))
```

Reply:
I have no idea what I was thinking there 😄